locales/en-US/faq.properties

```diff
@@ -2,38 +2,52 @@
+faq_item_7_paragraph_b=You can find more details about Mozilla’s expenditures and governance <a href="https://www.mozilla.org/foundation/documents/">here</a>
+faq_item_7_paragraph_a_thunderbird=We're the leading open source cross-platform email and calendaring client, free for business and personal use. Your donation will help ensure it stays that way, and will support future development.
 faq_item_8_header=Don't Mozilla products, like Firefox, earn income?
 faq_item_8_paragraph_a=Firefox is maintained by the Mozilla Corporation, a wholly-owned subsidiary of the Mozilla Foundation. While Firefox does produce revenue -- chiefly through search partnerships -- this earned income is largely reinvested back into the Corporation. The Mozilla Foundation's education and advocacy efforts, which span several continents and reach millions of people, are supported by philanthropic donations.
 faq_item_8_paragraph_b=Read more about the distinction <a href="http://mzl.la/1wSPI66">here</a>.
+faq_item_8_header_thunderbird=Doesn't Thunderbird earn income?
+faq_item_8_paragraph_a_thunderbird=Thunderbird is maintained by the Thunderbird Community, in cooperation with Mozilla Corporation, a wholly-owned subsidiary of the Mozilla Foundation. Thunderbird derives almost no revenue from search and other non-donor sources. Thunderbird relies heavily on the generous donations of users and friends of Thunderbird.
 faq_item_9_header=Who donates to Mozilla?
 faq_item_9_paragraph=Last year, more than 400,000 individuals gave to Mozilla to protect and improve the health of the internet. The Mozilla Foundation also receives support in the form of grants from other major foundations, like Knight, MacArthur and Ford.
+faq_item_9_header_thunderbird=Who donates to Thunderbird?
+faq_item_9_paragraph_thunderbird=Anyone can donate, companies and individuals, to Thunderbird to protect the product now and in the future. 
 faq_item_10_header=Does my donation give me access to tech support?
 faq_item_10_paragraph=Anyone can access tech support for Mozilla’s products by visiting the <a href="https://support.mozilla.org/questions">support forum</a>. Mozilla does not provide tech support or enhanced tech support in exchange for donation.
+faq_item_10_paragraph_thunderbird=Anyone can access tech support for Thunderbird by visiting the <a href="https://support.mozilla.org/products/thunderbird">support forum</a>. Thunderbird does not provide tech support or enhanced tech support in exchange for donation.
 faq_item_tax_header=Is my donation tax deductible?
 faq_item_tax_paragraph_a=The Mozilla Foundation is a California non-profit corporation exempt from United States federal income taxation under IRC 501(c)3, and donations are tax deductible for United States persons (legal aliens and citizens). The Mozilla Foundation Tax ID is 20-0097189.
```

Comment:
ok, we need an edit here to standardize tax-deductible lingo.  Can we please remove ... "The Mozilla Foundation is a California non-profit corporation exempt from United States federal income taxation under IRC 501(c)3, and donations are tax deductible for United States persons (legal aliens and citizens)." ...  and instead, insert "Contributions go to the Mozilla Foundation, a 501(c)(3) organization, to be used in its discretion for its charitable purposes. They are tax-deductible in the U.S. to the fullest extent permitted by law."  (Our lawyers do not want any variation on that language, at all.)

Reply:
I could do that in another issue, but better not touch it in this PR (it's not being added, just showing in context). How do we update to make localizers aware in this system? Bump the localization key?

Reply:
Yes, updating the key. I will take care of those in a separate PR that will get merged with this one